### PR TITLE
Removed the Optional<Ticket> method from the Ticket Repository

### PR DESCRIPTION
**Pull request by skamoen**
User can have multple Tickets, this will throw an exception when that is the case.

### CODE REVIEW

**Comment by TimvdLippe** [comment]
Could you add a regression test to prevent future problems?

**Comment by skamoen** [comment]
I removed the method. Does that count?

File: src/main/java/ch/wisv/areafiftylan/service/CurrentUserServiceImpl.java [hunk 1]
```diff
@@ -148,8 +147,9 @@ public boolean hasAnyTicket(Object principal) {
 
     @Override
     public boolean hasAnyTicket(String username) {
-        Optional<Ticket> ticket = ticketRepository.findByOwnerUsername(username);
-        return ticket.isPresent() && ticket.get().isValid();
+        Collection<Ticket> tickets = ticketRepository.findAllByOwnerUsername(username);
```
Comment (TimvdLippe) [inline src/main/java/ch/wisv/areafiftylan/service/CurrentUserServiceImpl.java]: Then I would turn this in a one-liner.

**Comment by TimvdLippe** [comment]
LGTM

**Pull request merged**